src/prefect/cli/worker.py

```diff
@@ -316,8 +316,9 @@ async def _get_worker_class(
             # Confirm with the user for installation in an interactive session
             elif install_policy == InstallPolicy.PROMPT and is_interactive():
                 message = (
-                    f"Could not find a {worker_type} worker in the current"
-                    " environment. Install it now?"
+                    "Could not find the Prefect integration library for the"
+                    " {worker_type} worker in the current environment."
```

Comment:
This needs to be an f-string, or else the output will literally be `{worker_type}`
```suggestion
                    f" {worker_type} worker in the current environment."
```